Prune unused requirements from requirements.txt

sampsyo opened the pull request:
These three libraries (awscli, elasticsearch, and sqlparse) don't seem to be used in the server code. bcrypt could also be removed based on the suggestion in #1.

sampsyo committed:
Remove awscli, elasticsearch, and sqlparse deps

These don't seem to be used in the server code.

sampsyo committed:
Add jwt dependency

sampsyo commented:
I also added a dependency on jwt, which was missing.

gvkarthik93 commented:
I have just pulled the changes. Didn't show conflict. Adding the dependency back.

sampsyo committed:
Fix jwt library name

Apparently, the JWT library the project uses is called `pyjwt` on PyPI,
not `jwt`, despite the module name.

SihanC reviewed (commented):
Updating jwt library name

SihanC committed:
Merge branch 'master' into prune-requirements
